diff --git a/td/telegram/Td.cpp b/td/telegram/Td.cpp
index 0cbcde9c..50e05ffa 100644
--- a/td/telegram/Td.cpp
+++ b/td/telegram/Td.cpp
@@ -4419,10 +4419,10 @@ Status Td::set_parameters(td_api::object_ptr<td_api::tdlibParameters> parameters
   if (options_.application_version.empty()) {
     return Status::Error(400, "Application version must be non-empty");
   }
-  if (options_.api_id != 21724) {
-    options_.application_version += ", TDLib ";
-    options_.application_version += TDLIB_VERSION;
-  }
+  // if (options_.api_id != 21724) {
+  //   options_.application_version += ", TDLib ";
+  //   options_.application_version += TDLIB_VERSION;
+  // }
   options_.language_pack = string();
   options_.language_code = string();
   options_.parameters = string();
